test/settings.cpp

```diff
@@ -110,7 +110,28 @@ BOOST_AUTO_TEST_CASE(settings__construct__mainnet_context__expected)
     BOOST_REQUIRE_EQUAL(configuration.net_sample, 1000);
     BOOST_REQUIRE_EQUAL(configuration.bip65_freeze, 388381);
     BOOST_REQUIRE_EQUAL(configuration.bip66_freeze, 363725);
+    BOOST_REQUIRE_EQUAL(configuration.bip34_freeze, 227931);
     BOOST_REQUIRE_EQUAL(configuration.bip16_activation_time, 0x4f779a80);
+    BOOST_REQUIRE_EQUAL(configuration.bip30_exception_checkpoint1,
```

Comment:
Style: don't wrap test lines of code. Use intermediate variables to shorten lines.

Reply:
Done.